pkg/plugins/scms/github/main.go

```diff
@@ -217,6 +226,11 @@ func New(s Spec, pipelineID string) (*Github, error) {
 		force = *s.Force
 	}
 
+	commitUsingApi := false
```

Comment:
I like that you set false by default, once the feature has been proved to be working well in production, I would imagine switching to true by default in a future release